src/main/java/reposense/model/LocationsCliArguments.java

```diff
@@ -13,14 +13,16 @@
 
     public LocationsCliArguments(List<String> locations, Path outputFilePath, Path assetsFilePath, Date sinceDate,
             Date untilDate, boolean isSinceDateProvided, boolean isUntilDateProvided, List<FileType> formats,
-            boolean isAutomaticallyLaunching, boolean isStandaloneConfigIgnored, ZoneId zoneId) {
+            boolean shouldIncludeLastModifiedDate, boolean isAutomaticallyLaunching, boolean isStandaloneConfigIgnored,
```

Comment:
Change the variable name to `isLastModifiedDateIncluded` to match the other variable name?